### DISCUSSION TITLE
Should jotai/query return more properties than just data?

### DISCUSSION THREAD

**Discussion by danielr18** [pmndrs/jotai#1424] [Ideas] [answered]
The `QueryObserverResult` has more properties than just data, that could be useful depending on what you are doing, such as `fetchStatus`.

https://tanstack.com/query/v4/docs/reference/useQuery

**Comment by dai-shi** [comment, marked as answer]
#1068 is the issue to track it.
After some discussion and consideration, it feels like we need to redesign an API, which will introduce some breaking changes.
I will work on it after #1404 which can be a base util.
